.github/workflows/deploy-worker.yaml

```
@@ -0,0 +1,29 @@
+    branches:
+      - 'main'
+    types: [closed]
+
+jobs:
+  deploy-to-cloudflare-workers:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: workers/images
+    outputs:
+      should_deploy: ${{ steps.check.outputs.should_run }}
```

actionlint (1.7.7)

property "check" is not defined in object type {}

(expression)